WordPress/Classes/ViewRelated/NUX/SiteCreation.storyboard

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is throwing a warning: `Unsupported Configuration: Prototype table cells must have reuse identifiers`.